documentation/jetty/modules/programming-guide/pages/server/http.adoc

```diff
@@ -537,6 +537,8 @@ The HTTP/3 client may now initiate a QUIC connection on port `843` and make HTTP
 
 NOTE: It is nowadays common to use the same port `443` for both HTTP/2 and HTTP/3. This does not cause problems because HTTP/2 listens on the TCP port `443`, while QUIC listens on the UDP port `443`.
 
+TIP: The `Alt-Svc` header is automatically added to HTTP/2 responses, advertising the HTTP/3 connector's port.
```

Comment:
Change `TIP` to `NOTE`, as this happens automatically, it is not a tip to developers.